Dockerfile

```diff
@@ -1,99 +1,131 @@
 # Base image
 FROM debian:bullseye AS base
-# Install OS Packages
+
+## Install OS Packages
 RUN apt update
-RUN apt install curl ca-certificates gnupg build-essential git jq -y
+RUN apt install curl ca-certificates gnupg build-essential git jq make g++ locales-all -y
```

Comment:
what are the extra packages for? I'm worried that this puts it over the 2GB default memory threshold and we have to add documentation so people recognize the error I was getting yesterday and need to increase it.

Reply:
These ones were already there, they are not new additions. I guess they are necessary, I just moved them from [line 14](https://github.com/iasql/iasql/pull/1954/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R14) to this `install` since these are general packages and those are `postgres` related packages

Reply:
non blocker, but do you think it makes sense to add a test that makes sure the size of the image doesn't increase or posts it in the PR or this would be overkill?

Reply:
a test might be overkill... We might be able to update the current message showing the install all modules time info to be more general giving us relevant info on the build process like time and size?

Reply:
> a test might be overkill... We might be able to update the current message showing the install all modules time info to be more general giving us relevant info on the build process like time and size?

that sounds good